test/e2e/node/node_problem_detector.go

```diff
@@ -233,3 +256,39 @@ func getCpuStat(f *framework.Framework, host string) (usage, uptime float64) {
 	usage *= 1e-9
 	return
 }
+
+func getNpdPodStat(f *framework.Framework, host string) (cpuUsage, rss, workingSet float64) {
+	summaryCmd := "curl localhost:10255/stats/summary"
```

Comment:
You can use https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/kubelet_stats.go#L285

Reply:
Done